python/ray/actor.py

```diff
@@ -841,7 +841,10 @@ def __getattribute__(self, attr):
         return object.__getattribute__(self, attr)
 
     def __repr__(self):
-        return "Actor(" + self._ray_actor_id.hex() + ")"
+        return "Actor({})".format(self._ray_actor_id)
```

Comment:
We probably want to keep the `.hex()`, right?

Reply:
ah fixed